compressai/utils/bench/codecs.py

```diff
@@ -394,7 +398,7 @@ def _set_args(self, args):
         return args
 
     def _get_encode_cmd(self, in_filepath, quality, out_filepath):
-        if not 0 <= quality <= 51:
+        if not 0 <= int(quality) <= 51:
```

Comment:
this is already an int here since `run` gets `quality: int`?

Reply:
`quality: int` doesn't convert the the of quality. But I know why it's str.  
It's not necessary here as `__main__.py` convert it to int.

```python
qps = [int(q) for q in args.qps.split(",") if q]
```

I removed it when I change `__main__.py`. But now it has been reverted.